app/models/grda_warehouse/config.rb

```diff
@@ -17,7 +19,7 @@ def self.cas_enabled?
 
     def self.relevant_state_codes
       # NOTE: memoized class variable, only changes on restart/redeploy
-      @relevant_state_codes ||= ENV['RELEVANT_COC_STATE']&.split(',') || ['MA']
+      @relevant_state_codes ||= GrdaWarehouse::Config.get(:relevant_state_codes)&.split(',') || ['MA']
```

Comment:
dont think ` || ['MA']` is necessary since the new column is non-nullable? (I realize this conflicts with my other question)

Reply:
That's probably true.  I was just going for minimal code changes.  I don't think it hurts, but I can remove if you think we should.